Twig/Extension/SonataAdminExtension.php

```diff
@@ -87,7 +112,13 @@ protected function getTemplate(FieldDescriptionInterface $fieldDescription, $def
             $template = $environment->loadTemplate($defaultTemplate);
 
             if (null !== $this->logger) {
-                $this->logger->warning(sprintf('An error occured trying to load the template "%s" for the field "%s", the default template "%s" was used instead: "%s". ', $templateName, $fieldDescription->getFieldName(), $defaultTemplate, $e->getMessage()));
+                $this->logger->warning(sprintf(
+                    'An error occured trying to load the template "%s" for the field "%s", '.
+                    'the default template "%s" was used instead.',
+                    $templateName,
+                    $fieldDescription->getFieldName(),
+                    $defaultTemplate
+                ), array('exception' => $e));
```

Comment:
@core23 @OskarStark BTW : this whole `$defaultTemplate` thing does not make much sense, don't you think. Removing it would be a b/c break, but I think we should remove that for 3.0